BeerMeUiKit/ViewController.swift

```diff
@@ -8,12 +8,79 @@
 import UIKit
 
 class ViewController: UIViewController {
```

Comment:
Questo nome è troppo generico, meglio utilizzare un nome che lasci intendere il contesto in cui è utilizzato